.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "vendor/asmcrypto.js"]
 	path = vendor/asmcrypto.js
 	url = https://github.com/vibornoff/asmcrypto.js
+[submodule "vendor/ethereumjs-wallet"]
```

Comment:
why submodule this in?

Reply:
I remember at the time that I could not find an equivalent npm package. Removed it and will use npm to include ethereumjs-wallet.